README.md

```diff
@@ -39,11 +40,22 @@ We follow the [Knowledge Project Guidelines](https://knowledge.github.io/knowled
 - Continuous integration testing with [Travis CI](https://about.travis-ci.com/)
 - Configurable build pipeline with support for custom build processes.
 - Scriptable deployment & migrations with Truffle framework.
+- Upgradable ERC20 Token contract.
 - Network management for deploying to many public & private networks.
 - Interactive console for direct contract communication.
 - Instant rebuilding of assets during development.
 - External script runner that executes scripts within a Truffle environment.
 
+## Contracts
+
+All ethereum solidity smart contracts can be found in the `/contracts` directory.
+
+- *Knowledge.sol*  is the ERC20 Token contract for the KNW token. It's an upgradable contract, which means that patches and upgrades can be deployed security without compromising the addresses, they get automatically migrated to the new contract version.
```

Comment:
This is actually not correct, the address will change if we deploy a new version of the contract. But token holders can migrate to the new contract address and still keep the same amount of tokens.